README.md

```diff
@@ -316,6 +317,31 @@ In all use-cases, the colon should be associated with the left-most item with no
 let myDictionary: [String: AnyObject] = ["String": 0]
 ```
 
+### Type Inference ###
+
+Prefer letting the compiler infer the type instead of explcitly stating it, wherever possible:
+
+```swift
+var max = 0 		// Int
+var name = "John" 	// String
+var rect = CGRect() // CGRect
```

Comment:
this is not supposed to be here. Xcode got confused. Maybe when you rebase it will refer to the right line again?